pyproject.toml

```diff
@@ -99,12 +107,12 @@ known_aiida = ['aiida']
 [tool.tox]
 legacy_tox_ini = """
 [tox]
-envlist = py38
+envlist = py39
```

Comment:
Is this the only environment we're wanting? Maybe I'm misreading this as I've not used `tox envlist`

Reply:
@federicazanca is currently working the tox framework for #41, so the intention here is to do the minimum update to be consistent, rather than the final set of versions.